tests/lib/nash/nash.go

```diff
@@ -64,7 +65,11 @@ func newNashShell(t *testing.T, output io.Writer) *nash.Shell {
 	shell.SetStdout(output)
 	shell.SetStderr(output)
 
-	nashPath := os.Getenv("HOME") + "/.nash"
+	dir, err := ioutil.TempDir("", "nash")
```

Comment:
we just keep creating temp dirs each time a test is run ? won't we clutter the filesystem with directories ? perhaps the test cleanup could delete this folder in the end of the execution.